Mars_QASpecFlow/Pages/ProfilePage.cs

```diff
@@ -0,0 +1,205 @@
+        IWebElement deleteIcon;
+        private readonly By skillButtonLocator = By.XPath("//*[@id='account-profile-section']/div/section[2]/div/div/div/div[3]/form/div[1]/a[2]");
+        IWebElement skillButton;
+        private readonly By addNewSkillButtonLocator = By.XPath("//*[@id='account-profile-section']/div/section[2]/div/div/div/div[3]/form/div[3]/div/div[2]/div/table/thead/tr/th[3]/div");
+        IWebElement addNewSkillButton;
+        private readonly By addBtnLocator = By.XPath("//*[@id='account-profile-section']/div/section[2]/div/div/div/div[3]/form/div[3]/div/div[2]/div/div/span/input[1]");
+        IWebElement addBtn;
+        private readonly By lastSkillLocator = By.XPath("//*[@id='account-profile-section']/div/section[2]/div/div/div/div[3]/form/div[3]/div/div[2]/div/table/tbody[last()]/tr/td[1]");
+        IWebElement lastSkill;
+        private readonly By editSkillButtonLocator = By.XPath("//*[@id='account-profile-section']/div/section[2]/div/div/div/div[3]/form/div[3]/div/div[2]/div/table/tbody[last()]/tr/td[3]/span[1]/i");
+        IWebElement editSkillButton;
+        private readonly By updateSkillButtonLocator = By.XPath("//*[@id='account-profile-section']/div/section[2]/div/div/div/div[3]/form/div[3]/div/div[2]/div/table/tbody[last()]/tr/td/div/span/input[1]");
+        IWebElement updateSkillButton;
+        private readonly By deleteSkillButtonLocator = By.XPath("//*[@id='account-profile-section']/div/section[2]/div/div/div/div[3]/form/div[3]/div/div[2]/div/table/tbody[last()]/tr/td[3]/span[2]/i");
+        IWebElement deleteSkillButton;
+
+        //Method to add new languages to the profile
+        public void AddNewLanguage(IWebDriver driver,string language,string level)
+        {
+            // Wait for some seconds
+            driver.Manage().Timeouts().ImplicitWait=TimeSpan.FromSeconds(5);
+            // Click on Add New button
+            addNewButton=driver.FindElement(addNewButtonLocator);
+            addNewButton.Click();
+            //Add Language
+            IWebElement addLanguage = driver.FindElement(By.Name("name"));
+            addLanguage.SendKeys(language);
+            //Add Language Level
+            IWebElement languageLevel = driver.FindElement(By.Name("level"));
+            languageLevel.SendKeys(level);
+            //Click on Add button
+            addButton = driver.FindElement(addButtonLocator);
+            addButton.Click();             
+        }
+
+        //Method to verify whether the languages are added to the profile
+        public string VerifyAddLanguage(IWebDriver driver,string language,string level)
+        {
+            driver.Navigate().Refresh();
+            languageName = driver.FindElement(languageNameLocator);
+            return languageName.Text;
+        }
+        //Method to edit languages in the profile
+        public void EditLanguages(IWebDriver driver,string editedLanguage,string editedLanguageLevel)
```

Comment:
4/
Please do not pass the driver In action methods (methods which in pages doing things like create, delete etc ) instead get the pages to extend to common driver where you will create a common driver class which will initialise the web driver 

public class CommonDriver
    {
        public static IWebDriver driver;

        public void initialise()
        {
            // create driver
            driver = new ChromeDriver();

        }

    }